xray/README.md

```diff
@@ -0,0 +1,55 @@
+# X-Ray
+
+A tool to inspect the performance of different wireguard implementations
+
+## How it works
+
+The setup consists of two clients and a wireguard interface. One of the clients sends and receives encrypted packets (let's call this the crypto socket), and the other sends and receives plaintext packets (let's call this the plaintext socket). The packets that are being sent contain a sequence number so that the receiving socket can now which packet it is, and a timestamp of when it was sent. 
```

Comment:
```suggestion
The setup consists of two sockets and a wireguard interface. One of the socket sends and receives encrypted packets (let's call this the crypto socket), and the other sends and receives plaintext packets (let's call this the plaintext socket). The packets that are being sent contain a sequence number so that the receiving socket can now which packet it is, and a timestamp of when it was sent. 
```

Reply:
Fixed